.github/workflows/run_tests_incremental.yml

```diff
@@ -98,13 +98,14 @@ jobs:
 
       - name: Test
         run: |
+          set -x
```

Comment:
This shouldn't leak the docker secret right, because github will strip it from the output?

Reply:
Right. Did a quick test on my fork to verify: https://github.com/sporksmith/shadow/runs/7653802389?check_suite_focus=true